src/sdk/conference/channel.js

```diff
@@ -876,25 +877,40 @@ export class ConferencePeerConnectionChannel extends EventDispatcher {
     const internalId = this._reverseIdMap.get(sessionId);
     if (this._subscribePromises.has(internalId)) {
       const mediaStream = this._remoteMediaStreams.get(sessionId);
-      const subscription = new Subscription(sessionId, mediaStream, () => {
-        this._unsubscribe(internalId);
-      }, () => this._getStats(),
-      (trackKind) => this._muteOrUnmute(sessionId, true, false, trackKind),
-      (trackKind) => this._muteOrUnmute(sessionId, false, false, trackKind),
-      (options) => this._applyOptions(sessionId, options));
+      const transportSettings =
+          new TransportSettings(TransportType.WEBRTC, this._id);
+      transportSettings.rtpTransceivers =
+          this._subscribeTransceivers.get(internalId).transceivers;
+      const subscription = new Subscription(
+          sessionId, mediaStream, transportSettings,
+          () => {
+            this._unsubscribe(internalId);
```

Comment:
resolve the subscribe promises here if stopped?

Reply:
This is the implementation of `subscription.stop()` function. Promise is resolved with the subscription here before `stop` function can be called.